### PR TITLE
Update APIs Related to Chrome 59

### PR DESCRIPTION
**Pull request by jpmedley**
New functionality in Chrome 59 adds items to these interfaces. I took this as an opportunity to fill out the details of these APIs.

### CODE REVIEW

File: macros/GroupData.json [hunk 1]
```diff
@@ -1333,14 +1346,41 @@
...
+                            "PresentationReceiver",
                             "PresentationRequest",
                             "PresentationSession",
                             "PresentationSessionConnectEvent" ],
-            "methods":    [],
+            "methods":    [ "PresentationConnection.close()",
+                            "PresentationConnection.send()",
+                            "PresentationConnection.terminate()",
+                            "PresentationRequest.start()",
+                            "PresentationRequest.reconnect()"
```
Comment (jwhitlock) [inline macros/GroupData.json]: [TravisCI noticed](https://travis-ci.org/mozilla/kumascript/builds/256945545#L1875-L1875) the missing comma.

**Comment by jpmedley** [comment]
Missing comma added. PTAL.

**Comment by chrisdavidmills** [comment]
The SW stuff looks good to me.

**Comment by jpmedley** [comment]
@a2sheppy Is there anything I can do to move this forward?

**Comment by jpmedley** [comment]
Then closing and resubmitting.

**Pull request closed without merging**